src/index.md

```diff
@@ -8,6 +8,10 @@ This is a free Rust course developed by the Android team at Google. The course c
 the full spectrum of Rust, from basic syntax to advanced topics like generics
 and error handling.
 
+> The latest version of the course can be found at
+> <https://google.github.io/comprehensive-rust/>. If you are reading
+> somewhere else, please check there for updates.
```

Comment:
I would be happy to hear better ideas for how to phrase this :smile:

Reply:
I think that's fine!

Reply:
Thanks!